libcore/src/IO/IniFileParser.cpp

```diff
@@ -283,13 +283,6 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
         // check file extension and if it is not matching the intended format,
         // change it to correct one
         switch(_config->GetFileFormat()) {
-            case FileFormat::XML: {
-                if(extension != ".xml") {
-                    trajectoryFile.replace_extension(".xml");
-                    LOG_WARNING("replaced output file extension with: .xml");
-                }
-                break;
-            }
             case FileFormat::TXT: {
```

Comment:
I just wanted to suggest removing this switch. But, maybe in the future we want to introduce a new format (binary)..

Reply:
I removed the switch, we could add it later on again.